Pcap++/src/PcapRemoteDevice.cpp

```diff
@@ -19,14 +19,14 @@ pcap_rmtauth PcapRemoteAuthentication::getPcapRmAuth() const
 	return result;
 }
 
-PcapRemoteDevice::PcapRemoteDevice(pcap_if_t* iface, PcapRemoteAuthentication* remoteAuthentication, const IPAddress& remoteMachineIP, uint16_t remoteMachinePort)
+PcapRemoteDevice::PcapRemoteDevice(pcap_if_t* iface, std::shared_ptr<PcapRemoteAuthentication> remoteAuthentication, const IPAddress& remoteMachineIP, uint16_t remoteMachinePort)
 	: PcapLiveDevice(iface, false, false, false)
+	, m_RemoteMachineIpAddress(remoteMachineIP)
```

Comment:
recommend the reorder of the construction list by the order of the arguments.

Reply:
It is currently ordered the way the members are declared in the class, which is the actual way they will be initialized.

Reply:
I see